src/components/index.js

```diff
@@ -13,5 +13,3 @@ export { default as Avatar } from "./Avatar";
 export { default as Badge } from "./Badge";
 export { default as Dropdown } from "./Dropdown";
 export { default as Button } from "./Button";
-
-export { default as Form } from "./Form";
```

Comment:
This will result in a naming clash as we would then have ./src/components/Form and ./src/forms/Form importing/exporting as Form in ./src/index.js so I've removed this for now